makeevrserg/Flipper-Android-App
# Fix infrared save

**Background**

Currently infrared feature has a wrong saving of infrared remotes. It may duplicate some time

**Changes**

- Fix insert key for inserting additional files also
- Fix navigation as configuring screen is not required

**Test plan**

- Open infrared remotes feature
- Try save file
- Try save file with same name
- See it now saved without duplicates and works better than before
